code/modules/admin/admin_verbs.dm

```diff
@@ -250,6 +251,7 @@ GLOBAL_LIST_INIT(admin_verbs_hideable, list(
 	/datum/admins/proc/startnow,
 	/datum/admins/proc/restart,
 	/datum/admins/proc/delay,
+	/datum/admins/proc/delay_round_end,
```

Comment:
tell me, why does `admin_verbs_hideable` exist?

Reply:
Genuinely no idea; would you prefer it be removed?

![image](https://user-images.githubusercontent.com/66640614/179435179-2ae44a46-a3bf-44c4-8133-c1209f639983.png)

![image](https://user-images.githubusercontent.com/66640614/179435192-bf5c159d-f4f0-41bf-a51e-466b616a0542.png)

![image](https://user-images.githubusercontent.com/66640614/179435220-cd0c5da4-f7cb-4129-a7c4-25f277959c63.png)

Reply:
yes